hashicorp/boundary-ui
# configure Desktop logging

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/JIRA_TICKET_NUMBER)

## Description

This PR installs `electron log` to add logs to desktop client.  

## Screenshots (if appropriate):

## How to Test

- run binary locally and perform search on DC 
- you can see the log file being generated in the below mentioned location based on your OS 

on Linux: ~/.config/{app name}/logs/main.log
on macOS: ~/Library/Logs/{app name}/main.log
on Windows: %USERPROFILE%\AppData\Roaming\{app name}\logs\main.log







:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
